models/Rooms.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python
```

Comment:
Instead of having a single file just for a fellow class or an office class, why don't you try having a Rooms file with the super class and the inheriting office and living space sub classes and a Person file with the respective subclasses?